sources/packages/backend/apps/db-migrations/src/main.ts

```diff
@@ -23,4 +23,6 @@ async function bootstrap() {
     await dataSource.destroy();
   }
 }
-bootstrap();
+(async () => {
```

Comment:
The code follows the Nestjs boilerplate.
If we need to change the `bootstrap` to be "auto-invoked" I believe that we may do it on the main method then, as shown below, or mark these ones as a false positive on Sonarcloud. Thoughts?
```ts
(async ( ) => {
  const migrationDataSource = new DataSource({
    ...ormConfig,
    logging: ["error", "warn", "info"],
  });
  const dataSource = await migrationDataSource.initialize();
  try {
    console.info("**** Running setupDB ****");
    await dataSource.query(`CREATE SCHEMA IF NOT EXISTS ${ormConfig.schema};`);
    await dataSource.query(`SET search_path TO ${ormConfig.schema}, public;`);
    await dataSource.query(`SET SCHEMA '${ormConfig.schema}';`);
    console.info(`**** Running migration ****`);
    await dataSource.runMigrations();
    console.info(`**** Running setupDB: [Complete] ****`);
  } catch (error) {
    console.error(`Exception occurs during setup db process: ${error}`);
    console.dir(ormConfig);
    throw error;
  } finally {
    await dataSource.destroy();
  }
})();
```

Reply:
That's fine having the IIFE, I think.

Reply:
I would incline to following the above suggested approach rather than marking these as false positives by Sonar

Reply:
ya, we can go with option 1